tests/multipleBackend/routes/routeBackbeatForReplication.js

```diff
@@ -1710,17 +1659,12 @@ describe('backbeat routes for replication', () => {
                 bucket: bucketSource,
                 objectKey: keyName,
                 queryObj: { versionId: 'null' },
-                authCredentials: destinationAuthCredentials,
+                authCredentials: sourceAuthCredentials,
```

Comment:
Why this change? It looks like we are on purpose getting the replicated null version as per the step name

Reply:
And the PUT into the `bucketDestination` is done below. If I try a GET on that bucket I get an `ObjNotFound`.

Now I think maybe we did not get any error in a zenko context because the `replication` credentials are using the zenko replication service account that can access every resources ?

Reply:
I've describe the test phases and renamed the steps to be clearer: [4e067fd](https://github.com/scality/cloudserver/pull/5919/commits/4e067fdbb908a498d9ab13c69d26d5300e466599)

The GET is done on the source to retrieve the object after lifecycle before replicating it to destination.

So the source credentials should be used on the source bucket